content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+## Bioactivity data profile
+
+[TOC]
+
+
+
+## Main Objective
+
+To provide guidance on the minimum set of metadata and semantics required to describe any chemical biology experiments, and how to translate bioactivity data to CHEMBL standards. This recipe also applies to binding, functional and ADME or toxicological assay types.
+
+
+## Graphical overview of the Recipe FAIRification Objectives
+
+```mermaid
+graph TD
+    A[Primary data at one concentration]-->B{Number of readouts per molecules?}
+    B-->|one|V{Which unit?}
+    V-->|percentage|D
+    V-->|value|F
+    B-->|multi|C[valid ones]
+    C-->|experiments|D{if triplicates}
+    D-->|aggregating or filtering|E[validated ones]
+    E--> |calculating|G[One average readout value <br> per concentration]
+    G-->F[CC/AC/IC50/IC90 curves]
+```
+
+### BioActivity data
+
+In the chemical biology domain (CBD), bioactivity data refers to datasets coming from high- to low-throughput biochemical and cellular screenings, hit confirmatory assay and profiling assays. Such datasets usually contain IC/AC/CC/EC50 (where 50 can be any number referring to the percent effect of the readout results). 
+
+Public CBD datasets are released in archives such as [ChEMBL](https://www.ebi.ac.uk/chembl/), [PubChem](https://pubchem.ncbi.nlm.nih.gov/), and [ECBD](https://ecbd.eu/).  They include general project- and sample-level information (like assay summaries or protocols), which will be briefly covered in this recipe but will also be covered in more depth elsewhere.
+
+A specific of this guideline is that they are referring mainly to ChEMBL formatting for uploading, so the covered suggested ontologies are intrinsically linked to “ChEMBL supported” ontologies ([https://www.ebi.ac.uk/ols/index](https://www.ebi.ac.uk/ols/index))
+
+
+#### Bioactivity data formats and standards
+
```

Comment:
Clarify if these formats and standards are requested by/used_by ChEMBL as part of deposition or "compatible with" ChEMBL deposition.